Formula/luyten.rb

```diff
@@ -28,10 +28,14 @@ def install
   end
 
   test do
-    io = IO.popen("luyten")
-    sleep 5
-    Process.kill("SIGINT", io.pid)
-    Process.wait(io.pid)
-    io.read !~ /Exception/
+    begin
+      io = IO.popen("#{bin}/luyten 2>&1")
+      sleep 5
+    ensure
+      Process.kill("SIGINT", io.pid)
+      Process.wait(io.pid)
+    end
+    output = io.read
+    assert_no_match /Exception/, output if output !~ /HeadlessException/
```

Comment:
So the typical test behavior is that we don't actually test anything? That seems like a lot of code for what essentially boils down to a no-op.

Not that I'm able to suggest a better test, if it's even possible to have a better test for what seems to be a pure GUI application—wait, didn't we have the homebrew/gui exile for these?

If it stays, I'd personally reduce this to:

```ruby
assert_predicate bin/"luyten", :exist?, "I'm an untestable GUI app."
```